docs/guides/multi-model.md

```diff
@@ -0,0 +1,108 @@
+# Multi-model endpoints
+
+_WARNING: you are on the master branch, please refer to the docs on the branch that matches your `cortex version`_
+
+It is possible to serve multiple models in the same Cortex API when using the Python predictor type (support for the TensorFlow predictor type is [coming soon](https://github.com/cortexlabs/cortex/issues/890)). In this guide, we'll deploy a sentiment analyzer and a text summarizer in one API, and we'll use query parameters to select the model. We'll be sharing a single GPU across both models.
+
+## Step 1: implement your API
+
+Create a new folder called "multi-model", and add these files:
+
+### `cortex.yaml`
+
+```yaml
+- name: text-analyzer
+  predictor:
+    type: python
+    path: predictor.py
+  compute:
+    cpu: 1
+    gpu: 1
+    mem: 12G
+  autoscaling:
+    threads_per_worker: 1
+```
+
+_Note: `threads_per_worker: 1` is the default, but setting it higher in production may increase throughput, especially when inference on one model takes a lot longer than the other(s)._
+
+### `requirements.txt`
+
+```python
+torch
+transformers==2.9.*
+```
+
+### `predictor.py`
+
+```python
+import torch
+from transformers import pipeline
+
+
+class PythonPredictor:
+    def __init__(self, config):
+        device = 0 if torch.cuda.is_available() else -1
+        print(f"using device: {'cuda' if device == 0 else 'cpu'}")
+
+        self.analyzer = pipeline(task="sentiment-analysis", device=device)
+        self.summarizer = pipeline(task="summarization", device=device)
+
+    def predict(self, query_params, payload):
+        if query_params.get("model") == "sentiment":
+            return self.analyzer(payload["text"])[0]
+        elif query_params.get("model") == "summarizer":
+            summary = self.summarizer(payload["text"])
+            return summary[0]["summary_text"]
+        else:
+            return {"error": f"unknown model: {query_params.get('model')}"}
```

Comment:
The error code would still be 200 for this error. We can showcase returning a starlette exception here or raise an exception.